### PR TITLE
[TP2-6] - create edit for user in userServices

### PR DESCRIPTION
**Pull request by concentration74**
A function was added to edit the user in the user service.

### CODE REVIEW

**Review by StayTeeVone — APPROVED** [review]
Good.

**Review by GOSTT22 — APPROVED** [review]

**Review by OleynikBogdan — APPROVED** [review]

File: src/app/custom-components/users.service.ts [hunk 1]
```diff
@@ -46,6 +46,13 @@ export class UserService {
             catchError(err => throwError('Invalid email or password.'))
         )
     }
+
+    updateUser(updatedUser: iUsers): Observable<any> {
+        return this.http.put<any>(this.url + '/' + updatedUser.id_user, updatedUser).pipe(
```
Comment (buba3230) [inline src/app/custom-components/users.service.ts]: Not any! You should pass an object with some type. Your interface type

File: src/app/custom-components/users.service.ts [hunk 2]
```diff
@@ -46,6 +46,13 @@ export class UserService {
...
+            tap(() => console.log('updated user')),
```
Comment (buba3230) [inline src/app/custom-components/users.service.ts]: Remove redundant tap from here

**Review by buba3230 — CHANGES_REQUESTED** [review]

**Review by GOSTT22 — APPROVED** [review]

**Review by StayTeeVone — CHANGES_REQUESTED** [review]
you know

**Review by StayTeeVone — APPROVED** [review]

**Review by OleynikBogdan — APPROVED** [review]

**Pull request closed without merging**